web/src/containers/BudgetSummary.js

```diff
@@ -12,25 +12,24 @@ const categoryLookup = {
   combined: 'Subtotal'
 };
 
-function DataRow({ data, title, year }) {
+function DataRow({ data, title }) {
   return (
-    <tr>
-      <th headers="summary-budget-null1 summary-budget-null2">{title}</th>
+    <tr
+      className={ title===categoryLookup.combined ? 'budget-table--subtotal budget-table--row__highlight' : '' }
```

Comment:
I'm sure there's a better way to do this than to generate a blank class, but I wasn't coming up with it.